test-harness/scenarios/operation-security-defined.oas3.scenario

```diff
@@ -0,0 +1,35 @@
+====test====
+Operation security defined, allow optional / no auth security
+====document====
```

Comment:
```suggestion
====asset:ruleset====
extends: [[spectral:oas, off]]
rules:
  oas3-operation-security-defined: error
```

As of recently, you can define local assets in a scenario, so let's leverage that feature and get rid of `test-harness/scenarios/rulesets/operation-security-defined.oas3.yaml`